test/service/RoutingService.test.js

```diff
@@ -18,6 +31,9 @@ describe('BvvRoutingService', () => {
 			expect(instanceUnderTest._categoriesProvider).toEqual(bvvRoutingCategoriesProvider);
 			expect(instanceUnderTest._routeProvider).toEqual(bvvRouteProvider);
 			expect(instanceUnderTest._routeStatsProvider).toEqual(bvvRouteStatsProvider);
+			expect(instanceUnderTest._chartItemsStylesProvider).toEqual(bvvChartItemStylesProvider);
+			expect(instanceUnderTest._mapper).toEqual(bvvOsmRoadTypeMappingProvider);
+			expect(instanceUnderTest._etaCalculationProvider).toEqual(bvvEtaCalculationProvider);
```

Comment:
Custom provider tests are missing in the next test `initializes the service with custom provider`